CODEOWNERS

```diff
@@ -1,4 +1,5 @@
 /actions/fleet @rancher/qa-pit-crew
+/actions/go.* @rancher/qa-pit-crew
```

Comment:
Can multiple teams be added here? It might be good to also add the rancher/raft-core team here also.